yew/src/agent/worker/private.rs

```diff
@@ -31,13 +38,30 @@ where
     type Agent = AGN;
 
     fn spawn_or_join(callback: Option<Callback<AGN::Output>>) -> Box<dyn Bridge<AGN>> {
+        let id = PRIVATE_ID_COUNTER.fetch_add(1, Ordering::Relaxed);
         let callback = callback.expect("Callback required for Private agents");
         let handler = move |data: Vec<u8>,
                             #[cfg(feature = "std_web")] worker: Value,
                             #[cfg(feature = "web_sys")] worker: &Worker| {
             let msg = FromWorker::<AGN::Output>::unpack(&data);
             match msg {
                 FromWorker::WorkerLoaded => {
+                    QUEUE.with(|queue| {
+                        queue.insert_loaded_agent(id);
+
+                        let mut msg_queue = queue.borrow_msg_queue_mut();
+                        if let Some(msgs) = msg_queue.get_mut(&id) {
+                            for msg in msgs.drain(..) {
+                                cfg_match! {
+                                    feature = "std_web" => ({
+                                        let worker = &worker;
+                                        js! {@{worker}.postMessage(@{msg});};
+                                    }),
+                                    feature = "web_sys" => worker.post_message_vec(msg),
+                                }
+                            }
+                        }
+                    });
                     send_to_remote::<AGN>(&worker, ToWorker::Connected(SINGLETON_ID));
```

Comment:
We want this message to be sent before any of the other queued messages. It can actually be added to the message queue as soon as the bridge is created.

Reply:
But this is line `65` and the bridge is created in line `92`; The public worker creates the bridge first on the other hand.

Reply:
Hmm why can't you call `bridge.send_message(..)` right after the bridge is created?

Reply:
I can, but that'd be later, you were saying it should be sent before any of the other messages.

Reply:
Can't get any earlier than right after the bridge is created

Reply:
As far as I can tell, right now we're draining the queue first, then we send this message, then we initalize the worker on line `76` (`52` before this PR), and only then we initalize the bridge on line `92` (`68` before the PR)

You have been saying to move the `send_to_remote::<AGN>(&worker, ToWorker::Connected(SINGLETON_ID));` call ahead of the other messages sent (i.e. ahead of emptying the queue), but now you are also saying that it should be done right after the bridge is created, so I am confused about that!

Unless I misunderstand how the bridge is created in the first place, is it not initalized on the line I specified above?

Reply:
You are correct about where the bridge is created but note that the queue is drained in the handler callback and won't be drained until the worker is loaded. Look more closely at the code please to understand the order in which things are executed